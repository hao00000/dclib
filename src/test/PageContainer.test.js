import React from 'react'
import {mount} from 'enzyme'
import TurnstileContainer from '../demo/pageContainers/turnstile/TurnstileContainer'

describe('Page container test', () => {
  let component
  beforeAll(done => {
    component = mount(<TurnstileContainer.TurnstileContainer />)
    setTimeout(done, 5)
  })

  it('should render the component', () => {
    expect(component.html()).toMatchSnapshot()
  })
})
