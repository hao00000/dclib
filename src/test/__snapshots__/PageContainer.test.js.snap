// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page container test should render the component 1`] = `"<div class=\\"form-view\\"><p>This is a sample page for CUSTOM_PAGE type.</p><p>This page is loaded through PageContainer setting, see demo/index.js and appConfig.json</p></div>"`;
